libclamav_rust/build.rs

```diff
@@ -157,7 +159,10 @@ fn execute_bindgen() -> Result<(), &'static str> {
     }
 
     for &include_path in BINDGEN_INCLUDE_PATHS {
-        builder = builder.clang_arg(include_path);
+        builder = builder
+            .clang_arg(include_path)
+            // Hack to include 3rd party C libary headers
+            .clang_arg("-I/home/micah/.mussels/install/host-static/include");
```

Comment:
Looks like you forgot to fix this?

Reply:
Oh, crap. I absolutely need to remove this.  I think this was a workaround for issues with having openssl-1.1-dev installed to my system while building with openssl 3.  It's not relevant to this PR at all and should never have been committed.

Reply:
Fixed.